### PR TITLE
RTB-6 Added Password copying into Clipboard and added Toasters

### PR DESCRIPTION
**Pull request by folivora97**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@fortawesome/fontawesome-free": "^6.4.0",
     "admin-lte": "^4.0.0-alpha2",
     "bootstrap": "^5.3.0",
-    "jquery": "^3.7.0"
+    "jquery": "^3.7.0",
+    "toastr": "^2.1.4"
```
Comment (Relluem94) [inline package.json:24]: toastr is unmaintained 
image

```JavaScript
var msg = 'Folivora <script>alert("DangerDan!")</sc' + 'ript>'; 
toastr.success("Exploit <strong>Message</strong> " + msg);

File: tools/passwordGenerator.php [hunk 2]
```diff
@@ -10,9 +12,28 @@ function generatePassword() {
     $("#password")[0].value = password;
 }
 
+
+async function copyToClipboard(){
```
Comment (Relluem94) [inline tools/passwordGenerator.php:16]: image
image

do you get the same error? i'm using firefox@fedora & chrome@fedora

**Review by Relluem94 — CHANGES_REQUESTED** [review]
i like the idea of toasters but we have to find an alternative to toastr as it is no longer maintained.

File: package.json (hunk 1 above)
Comment (folivora97) [reply package.json:24]: We will switch out with bootstraps native toasters :)

File: tools/passwordGenerator.php (hunk 2 above)
Comment (folivora97) [reply tools/passwordGenerator.php:16]: You need to grant Permissions for Clipboard in Chrome, like for Camera or Microphone Usage

File: package.json (hunk 1 above)
Comment (folivora97) [reply package.json:24]: Changed to own baked Toast. Yummy.

**Review by Relluem94 — APPROVED** [review]

File: assets/folitoast/folitoast.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    height: 100px;
+    position: fixed; 
+    top: 0px; 
+    right: 0px;
+    margin-right: 20px;
+    margin-top: 20px;
+    padding: 20px;
+    font-family: Arial, Helvetica, sans-serif;
+    color: white;
+    font-size: 15px;
```
Comment (Relluem94) [inline assets/folitoast/folitoast.css:14]:
```suggestion
    font-size: 15px;
    z-index:9999999999999;
```

**Review by Relluem94 — APPROVED** [review]

**Pull request merged**